typescript/ec2-instance/lib/ec2-cdk-stack.ts

```diff
@@ -2,21 +2,21 @@ import * as ec2 from "aws-cdk-lib/aws-ec2";
 import * as cdk from 'aws-cdk-lib';
 import * as iam from 'aws-cdk-lib/aws-iam'
 import * as path from 'path';
-// import { KeyPair } from 'cdk-ec2-key-pair';
+import { KeyPair } from 'cdk-ec2-key-pair';
 import { Asset } from 'aws-cdk-lib/aws-s3-assets';
 import { Construct } from 'constructs';
 
+
 export class Ec2CdkStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props?: cdk.StackProps) {
     super(scope, id, props);
 
     // Create a Key Pair to be used with this EC2 Instance
-    // Temporarily disabled since `cdk-ec2-key-pair` is not yet CDK v2 compatible
-    // const key = new KeyPair(this, 'KeyPair', {
-    //   name: 'cdk-keypair',
-    //   description: 'Key Pair created with CDK Deployment',
-    // });
-    // key.grantReadOnPublicKey
+    const key = new KeyPair(this, 'KeyPair', {
+      name: 'cdk-keypair',
```

Comment:
I would request that this be randomized (I believe a random name will be generated if you don't provide a name) and provide an Output for it.